engines/scumm/scumm.cpp

```diff
@@ -2759,7 +2758,10 @@ bool ScummEngine::startManiac() {
 		eventMan->pushEvent(event);
 		return true;
 	} else {
-		displayMessage(0, "%s", _("Usually, Maniac Mansion would start now. But for that to work, the game files for Maniac Mansion have to be in the 'Maniac' directory inside the Tentacle game directory, and the game has to be added to ScummVM."));
```

Comment:
all this did, was go into displayMessage, use vsnprintf, and call MessageDialog. to make translations work, I just shifted it out and used u32::format